OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -207,7 +207,7 @@ public string VoicePhraseForOrder(Actor self, Order order)
 			if (order.OrderString != "Unload" || IsEmpty(self))
 				return null;
 
-			return self.HasVoice("Unload") ? "Unload" : "Move";
+			return self.TraitsImplementing<IVoiced>().Any(x => x.HasVoice(self, "Unload")) ? "Unload" : "Move";
```

Comment:
This is the only trait that hardcodes a fallback to "Move".  How about we fix the voice sets and remove this?